packages/dd-trace/test/appsec/index.express.plugin.spec.js

```diff
@@ -46,11 +45,10 @@ withVersions('express', 'express', version => {
 
       app.param('callbackedParameter', paramCallbackSpy)
 
-      getPort().then((port) => {
-        server = app.listen(port, () => {
-          axios = Axios.create({ baseURL: `http://localhost:${port}` })
-          done()
-        })
+      server = app.listen(0, () => {
```

Comment:
Nit: Specifying no port and specifying `0` is the same in Node.js

```suggestion
      server = app.listen(() => {
```